secv_guis/base_widgets.py

```diff
@@ -48,19 +48,22 @@ def __init__(self, label, parent=None,
         self.file_watcher.directoryChanged.connect(
             lambda: self.update_path(self.dirpath))
 
-    def update_path(self, dirname,file_filter=None):
+    def update_path(self, dirname,selected_image=None):
         """
         :param str dirname: The new directory path to be listed.
+        :param selected_images: The name of an image with which the list can be filtered
```

Comment:
The name is `selected_image` not `selected_images`

Reply:
ToDo